src/models/disassemblyoutput.cpp

```diff
@@ -43,15 +43,24 @@ static QVector<DisassemblyOutput::DisassemblyLine> objdumpParse(const QByteArray
         if (asmLine.isEmpty() || asmLine.startsWith(QLatin1String("Disassembly")))
             continue;
 
+        const int colonIndex = asmLine.indexOf(QLatin1Char(':'));
+        const int angleBracketIndex = asmLine.indexOf(QLatin1Char('<'));
+
+        // skip function header line as it is not interesting for us
```

Comment:
can you give an example for such a line?
doesn't this break for symbol names like

Foo::Bar<Asdf>()

Reply:
the lines look like this `0000000000001090 <main>:` of `0000000000001264 <_ZN4Test4testEv>:`
the names are mangled so it not a problem